app/docs/0.13.x/getting-started/adding-your-api.md

```diff
@@ -12,61 +12,94 @@ title: Adding your API
   </ol>
 </div>
 
-In this section, you'll be adding your API to the Kong layer. This is the first
+In this section, you'll be adding your Service to the Kong layer. This is the first
```

Comment:
You changed this file to "configuring-a-service.md", this adding-your-api.md file should be deleted.